### PR TITLE
De45353/fix face header hierarchy

### PR DESCRIPTION
**Pull request by ronan-f**
https://rally1.rallydev.com/#/29180338367d/custom/486232622040?detail=%2Fdefect%2F609974365545

The goal with this change is to make the heading hierarchy accessible in FACE. The lang terms will potentially change but design has approved them for now. Refer to the screenshots below for the desired hierarchy. I'm not a big fan of adding the same lang terms in multiple files but due to the fact that different localize mixins are being used in the necessary files I'm not sure we have an easy way to share them. 

I tested this using NVDA and everything seemed to get announced correctly. 

The change for the immersive nav bar can be found here: https://github.com/Brightspace/lms/pull/16670

image

image

### CODE REVIEW

**Review by dlockhart — DISMISSED** [review]
Looks good. If you did want to reduce the duplication of the lang terms and the markup (which I'm assuming will get worse as more and more activity types are converted), you could create a couple shared components -- e.g. `<d2l-activity-editor-core-properties-heading>` and `<d2l-activity-editor-advanced-properties-heading>`.

**Comment by ronan-f** [comment]
> Looks good. If you did want to reduce the duplication of the lang terms and the markup (which I'm assuming will get worse as more and more activity types are converted), you could create a couple shared components -- e.g. `<d2l-activity-editor-core-properties-heading>` and `<d2l-activity-editor-advanced-properties-heading>`.

Lol that's a 🤦  moment. Making a component is much more straightforward than what I was thinking... Thanks Dave.

File: components/d2l-activity-editor/d2l-activity-quiz-editor/lang/en.js [hunk 1]
```diff
@@ -243,5 +243,5 @@ export default {
 	"submissionViewsDialogSecondaryEditorGradeRangeTextAnd" : "and", // text in submission view secondary view edit view, grade bound text between lower and upper amounts
 	"submissionViewsDialogSecondaryEditorNumericalInputText" : "minute(s)", // text in submission view secondary view edit view, text after the time limit box.
 	"submissionViewsDialogSecondaryEditorTimeLimitText": "Time Limit", // text in submission view secondary view edit view, text above the time limit box.
-	"newQuizCategory": "New Quiz Category" // new quiz category dialog title
+	"newQuizCategory": "New Quiz Category", // new quiz category dialog title
```
Comment (dlockhart) [inline components/d2l-activity-editor/d2l-activity-quiz-editor/lang/en.js:246]: Nit: I think this is still valid JS, but if you want to make the diff from this PR smaller you could remove it. There's a similar whitespace change in the other lang file as well.

**Review by dlockhart — DISMISSED** [review]
Awesome!

**Review by dlockhart — APPROVED** [review]

**Pull request merged**